Bottom: c4906b969c5b4cb5a5581ea26ac771988a8df770
Top:    e1d59c034ebe1eb78777072087b7616acd3fecaf
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-09-05 12:53:53 +0100

hid/gtk: Turn on defines to debug a potential GTK-3.0 build


---

diff --git a/configure.ac b/configure.ac
index 71ab1c4..1150f22 100644
--- a/configure.ac
+++ b/configure.ac
@@ -770,6 +770,9 @@ $GTKGLEXT_PKG_ERRORS])]
 	GTKGLEXT_VER=`$PKG_CONFIG gtkglext-1.0 --modversion`
 	fi
 
+	GTK_CFLAGS="$GTK_CFLAGS -DGSEAL_ENABLE"
+	GTK_CFLAGS="$GTK_CFLAGS -DGDK_DISABLE_DEPRECATED -DGTK_DISABLE_DEPRECATED"
+
 	;;
 
       gcode|nelma|png )
